app/views/layouts/application.html.erb

```diff
@@ -89,12 +89,12 @@
 
 	<div id="global-app-error" class="app-error hidden"></div>
 
-	<script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
+	<script src="//ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
 
 	<script>window.jQuery || document.write('<script src="/assets/jquery.min.js">\x3C/script>')</script>
 	<%= javascript_include_tag "application" %>
 	<%- if admin_area? %>
-		<script src="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.0/jquery-ui.min.js"></script>
+		<script src="//ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
```

Comment:
IMHO JS is a mess in this app. I would like to spend some time in the future to refactor these bits. But in the meantime, I will check if we can rely on application js.